server/src/database/transaction_schema.graphql

```diff
@@ -0,0 +1,40 @@
+"An ISO-8601 encoded UTC date string."
+scalar DateTime
+
+type Query {
+  transactions(query: TransactionQueryInput, limit: Int = 100, sortBy: TransactionSortByInput): [Transaction]!
+}
+
+type Transaction {
+  amount: Float
+  canonical: Boolean
+  memo: String
+  dateTime: String
+  hash: String
+  blockHeight: Int
+  kind: String
+  to: String
+  from: String
+  none: Int
```

Comment:
This should be `nonce` instaed of `none`

Reply:
definitely! Fixed https://github.com/Granola-Team/mina-on-chain-voting/pull/237/commits/68391f706ab87cb2d43722098d58739897d7a61e